docs/components/modeler/dmn/dmn.md

```diff
@@ -0,0 +1,62 @@
+---
+id: camunda-modeler-dmn
+title: Editing DMN in Camunda Modeler
+description: To start modeling, let's create a new DMN diagram.
+---
+
+<span class="badge badge--CCSM">CCSM</span>
```

Comment:
This is actually only available in Platform, so could you make a badge for "Camunda Platform Only". 

Given how small and not obvious the badging is for this, can you also add an admonition? Something like "DMN is only available in Camunda Platform at this time."